packages/components/src/lib/utility.ts

```diff
@@ -120,3 +120,9 @@ export function adjustColorBrightness(color: string, amount: number) {
       )
   );
 }
+
+export type SqValueWithPath = SqValue & Required<Pick<SqValue, "path">>;
```

Comment:
maybe add a comment on top explaining, for people without much knowledge of the syntax.
"Ensures that the SqValue has a "path" parameter, which would otherwise be optional"